feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.spec.ts

```diff
@@ -292,6 +292,38 @@ describe('ConfiguratorAttributeDropDownComponent', () => {
       );
     });
 
+    it('should display quantity and no price', () => {
+      component.attribute.quantity = 5;
+      component.attribute.attributePriceTotal = {
+        currencyIso: '$',
+        formattedValue: '500.00$',
+        value: 500,
+      };
+
+      let value = component.attribute.values
+        ? component.attribute.values[0]
+        : undefined;
+      if (value) {
+        value.valuePrice = undefined;
+      } else {
+        fail('Value not available');
+      }
+
+      fixture.detectChanges();
```

Comment:
Line 296-312 are not needed. Please see the test data setup, the first value is a retract one.